querybook/webapp/const/board.ts

```diff
@@ -1,32 +1,40 @@
+import { ContentState } from 'draft-js';
 import { IDataDoc } from './datadoc';
 import { IDataTable } from './metastore';
 
-export interface IBoardRaw extends IBoard {
-    docs: IDataDoc[];
-    tables: IDataTable[];
-    items: IBoardItem[];
-}
-
-export interface IBoardWithItemIds extends IBoard {
-    docs: number[];
-    tables: number[];
-    items: number[];
-}
-
-export interface IBoard {
+export interface IBoardBase {
     id: number;
     created_at: number;
     updated_at: number;
     deleted_at: number;
 
     name: string;
-    description: string;
+    description: string | ContentState;
```

Comment:
description can just be string here

Reply:
that errors

Interface 'IBoard' incorrectly extends interface 'IBoardBase'.
  Types of property 'description' are incompatible.
    Type 'ContentState' is not assignable to type 'string'

Reply:
https://stackoverflow.com/questions/41285211/overriding-interface-property-type-defined-in-typescript-d-ts-file